src/common/util/time-cache-function-promise.ts

```diff
@@ -7,14 +7,10 @@ interface ResultCache<T> {
 export const timeCachePromiseFunc = async <T>(
   cacheKey: string,
   cacheTime: number,
-  func: (
-    hass: HomeAssistant,
-    entityId: string,
-    ...args: unknown[]
-  ) => Promise<T>,
+  func: (hass: HomeAssistant, entityId: string, ...args: any[]) => Promise<T>,
   hass: HomeAssistant,
   entityId: string,
-  ...args: unknown[]
+  ...args: any[]
```

Comment:
It is working with `width` and `height` but I had to change the typing here.  Not sure this it the best solution.